aspnet-core/src/Timesheet.Application/APIs/ReviewDetails/ReviewDetailAppService.cs

```diff
@@ -1354,6 +1354,33 @@ public async Task HeadPmVerify(HeadPmVerifyDto input)
                 throw new UserFriendlyException("Trạng thái chỉ được là Reviewed hoặc Rejected.");
             }
         }
+            
+        [AbpAuthorize(Ncc.Authorization.PermissionNames.ReviewIntern_ReviewDetail_AcceptPMReviewForAllIntern)]
+        [HttpPost]
+        public async Task HeadPmVerifyOrRejectAll(List<HeadPmVerifyDto> input)
+        {
+            foreach(var inputItem in input)
+            {
+                if (inputItem.Status == ReviewInternStatus.Reviewed || inputItem.Status == ReviewInternStatus.Rejected)
+                {
+                    var detail = await WorkScope.GetAsync<ReviewDetail>(inputItem.ReviewDetailId);
+                    if (detail.Status == ReviewInternStatus.PmReviewed)
+                    {
+                        detail.Status = inputItem.Status;
+                        await WorkScope.UpdateAsync(detail);
+                    }
+                    else
+                    {
+                        throw new UserFriendlyException("Review này chưa PM Review");
+                    }
+                }
+                else
+                {
+                    throw new UserFriendlyException("Trạng thái chỉ được là Reviewed hoặc Rejected.");
+                }
+            }
+            
```

Comment:
This method needs to check if the framework supports saving bulk items -> we should use save bulk instead.
Need to update the error message to consistent with the language on the FE.